ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -147,6 +147,51 @@
       }
     ]
   },
+  {
```

Comment:
auth for system update user should really be defined here now to have one source of truth

Reply:
done